src/layouts/DataImportBatch__c-NPSP Data Import Batch Layout.layout

```diff
@@ -173,7 +107,7 @@
     <showRunAssignmentRulesCheckbox>false</showRunAssignmentRulesCheckbox>
     <showSubmitAndAttachButton>false</showSubmitAndAttachButton>
     <summaryLayout>
-        <masterLabel>00hf100000Apz2c</masterLabel>
+        <masterLabel>00hZ0000000pFXt</masterLabel>
```

Comment:
Hard coded Ids concern me. Should this?

Reply:
Same, but it is something we have used previously, such as with the Filter_Rule__mdt page layout: https://github.com/SalesforceFoundation/Cumulus/blob/3223a213e3708d0ffefa5451dee53dd684a57725/src/layouts/Filter_Rule__mdt-Filter%20Rule%20Layout.layout

Reply:
Also confirmed this is from the platform: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_layouts.htm